docs/kubeapps-on-bkpr.md

```diff
@@ -0,0 +1,90 @@
+# Installing Kubeapps on BKPR
+
+This guide documents the installation of [Kubeapps](https://kubeapps.com/) on the Kubernetes Production Runtime (BKPR) cluster. You will first install the [Helm](https://www.helm.sh/) package manager to the cluster, followed by the installation of Kubeapps using the [Kubeapps Helm chart](https://hub.kubeapps.com/charts/bitnami/kubeapps).
+
+The guide assumes you have a Kubernetes cluster with BKPR already installed. The [BKPR installation guide](install.md) documents the process of installing the BKPR client, followed by installing the BKPR server side components on the cluster.
+
+## Step 1: Install the Helm client
```

Comment:
Instead of linking to Helm docs, do you want to link to our docs instead? We cover all this in https://docs.bitnami.com/kubernetes/get-started-kubernetes/#step-4-install-and-configure-helm-and-tiller

Reply:
...but check once that it covers all your needs

Reply:
The linked docs do not document installing tiller for secure access. I would leave this as-is.